### ISSUE TITLE
Autoscaling for all diagram types

### ISSUE THREAD

**Issue by knsv** [knsv/mermaid#175]
It is not uncommon when integrating mermaid on a documentation site that some of the diagrams in the generated documentation is wider then the allocated width for the content.

As the diagrams are svgs they could scale instead of flowing over.

However it would be undesirable to have a the diagram grow in width. Imaging a single box in a flowchart diagram autoscaling to 1200px. So the diagrams should only scale to zoom out.

**Comment by knsv** [comment]
As this could potentially make svgs less functional outside the browser the new functionality can be turned off with a new configuration parameter:
```
flowchart.useMaxWidth: [true|false]
sequenceDiagram.useMaxWidth:[true|false]
```

**Comment by knsv** [comment]
Fixed and pushed into master. Will be included in 0.5.1

**Comment by knsv** [comment]
Fixed with release 0.5.1
